platform-common-core/src/test/java/com/softicar/platform/common/core/number/formatter/BigDecimalFormatterTest.java

```diff
@@ -0,0 +1,77 @@
+package com.softicar.platform.common.core.number.formatter;
+
+import com.softicar.platform.common.core.locale.ILocale;
+import com.softicar.platform.common.core.locale.Locale;
+import java.math.BigDecimal;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BigDecimalFormatterTest extends Assert {
```

Comment:
This test does not, in any way, ensure that `BigDecimalFormatter` uses `CurrentLocale`. However, this is likely to be the most common use case.

We could even manipulate `BigDecimalFormatter`, replacing
`this.locale = CurrentLocale.get();`
with
`this.locale = null`
The thing would crash (unless `setLocale` gets called) - yet all tests will still be green.

Reply:
did so; to made this testable, I introduced new `LocaleScope`